### PR TITLE
feat: added plugin that allows you to focus windows in Hyprland

### PR DESCRIPTION
**Pull request by abenz1267**
Let me know what you think!

Not much to say from my side... pretty straight forward.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -5,13 +5,14 @@ members = [
...
+  "plugins/hyprlandwindows",
   "plugins/kidex",
-  "plugins/translate",
   "plugins/randr",
+  "plugins/rink",
+  "plugins/shell",
   "plugins/stdin",
-  "plugins/dictionary",
+  "plugins/symbols",
+  "plugins/translate",
```
Comment (Sntx626) [inline Cargo.toml:16]: I though about alphabetically sorting the entries myself ^^
Thanks

**Review by Sntx626 — APPROVED** [review]
Looking good!
Though I have to say that I'm not running hyprland myself, so I couldn't test it.

**Comment by abenz1267** [comment]
I've added a prefix option.

**Comment by Kirottu** [comment]
A neat idea and a useful plugin, but I think I'll need to start reconsidering how community contributed plugins are added to this project.

Having them all in the same repo is convenient, but also a bit of a burden maintainment wise especially as I may not be familiar with the codebase.

Do any of you have any ideas on how to deal with this? Or ideas about something like having proper versioning that I should have dealt with a long time ago?

**Comment by Sntx626** [comment]
I think we should open up an Issue for that.

**Comment by abenz1267** [comment]
Yeah, i can definitely see how this could be a burden. But i'm not totally sure if there's a better way.

The most sane option in regards to maintance would be to not have community contributed plugins in the main repo, but maybe just link to them from the readme or something.

You'd loose the convenience of automatically installing potentially "basic" plugins via Nix/AUR then, though.

Maybe start a discussion, so we can continue the conversation there.

**Comment by Kirottu** [comment]
> I think we should open up an Issue for that.

I agree, I'll create a pinned issue for this. I have a few ideas on how it could be handled.

**Comment by abenz1267** [comment]
I just noticed someone already created a plugin for Hyprland Windows:

https://github.com/Kirottu/anyrun/issues/22

**Pull request closed without merging**